<?xml version="1.0" encoding="UTF-8" ?>
<testsuite errors="0" failures="1" hostname="AmIts-MacBook-Pro.local" name="auctionhaus.CustomerIntegrationTestTests" tests="1" time="0.32" timestamp="2012-02-15T06:16:47">
  <properties />
  <testcase classname="auctionhaus.CustomerIntegrationTestTests" name="testUniqueEmail" time="0.319">
    <failure message="Assertion failed: &#xa;&#xa;assert &apos;unique&apos; == customer.errors[&quot;email&quot;]&#xa;                |  |        |     |&#xa;                |  |        |     null&#xa;                |  |        grails.validation.ValidationErrors: 0 errors&#xa;                |  auctionhaus.Customer : 1&#xa;                false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert &apos;unique&apos; == customer.errors[&quot;email&quot;]
                |  |        |     |
                |  |        |     null
                |  |        grails.validation.ValidationErrors: 0 errors
                |  auctionhaus.Customer : 1
                false

	at auctionhaus.CustomerIntegrationTestTests.testUniqueEmail(CustomerIntegrationTestTests.groovy:30)
</failure>
  </testcase>
  <system-out><![CDATA[--Output from testUniqueEmail--
]]></system-out>
  <system-err><![CDATA[--Output from testUniqueEmail--
]]></system-err>
</testsuite>
